samples/custom_dispatch/cpu/mlp_plugin/mlp_spec.tosa.pdl.mlir

```diff
@@ -0,0 +1,117 @@
+// PDL pattern spec to match an MLP and offload to an external function
```

Comment:
`mlp_tosa_spec.pdl.mlir` or something (given that there's more files growing in this folder, keeping the prefixes the same so they sort and it's clear what's related would be nice - `mlp_tosa.mlir` + `mlp_tosa<whatever>`)